feast-acquisition-events.cloudformation.yaml

```
@@ -0,0 +1,304 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: FeastGoogleAcquisitionEventsLambda
```

I don't think this is Google specific now, right?

```suggestion
Description: FeastAcquisitionEventsLambda
```

---

Updated in latest commit.